.github/workflows/create-release.yml

```diff
@@ -56,7 +56,8 @@ jobs:
     - name: Setup node
       uses: actions/setup-node@v4
       with:
-        node-version: latest
+        # renovate: datasource=node-version depName=node
+        node-version: 20
```

Comment:
Makes tests more repeatable. Should we change this to include minor versions? Minor versions can sometimes include major NPM updates IIRC.

Patch versions are released too often, would result in spam

Reply:
action now has a node-version-file input, I'll use that instead.